source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/customcontrols/ValidatedTextView.java

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
every new added file should be a kotlin file

Reply:
updated